install.sh

```diff
@@ -0,0 +1,165 @@
+#!/bin/bash
+
+# CHECKS FOR SUDO ACCESS (added because of those who have seperate partitions for /home and /root) 
+
+if [ "$EUID" -ne 0 ]
+  then echo "Please don't run as root" && exit
+  exit
+fi
+
+# MAKE DIRECTORY SO THAT YOUR HOME DIRECTORY DOESN'T GET DIRTY
+
+cd /home/$USER
+mkdir -p junk-drawer
+cd junk-drawer
+
+# Install git and wget
+
+echo "What distro do you use?"
```

Comment:
We could do this automatically, I.E
```sh
cat /etc/issue | grep -Er (insert regex here, I'm not sure the format of /etc/issue lol)

Reply:
Ok, so i found a way to find out the users distro automatically.
```
export DISTRO=$(echo "$(lsb_release -is)")
```

Does that look good?